helper/acctest/skip_on_error_test.go

```diff
@@ -0,0 +1,25 @@
+package acctest
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestSkipOnError(t *testing.T) {
+	testCases := []struct {
```

Comment:
These unit test cases should include a negative match. 👍

Reply:
Added!